crypto/asn1/a_object.c

```diff
@@ -85,7 +85,7 @@ int a2d_ASN1_OBJECT(unsigned char *out, int olen, const char *buf, int num)
             c = *(p++);
             if ((c == ' ') || (c == '.'))
```

Comment:
But the new code is (below). The `isdigit()` below this was changed, implying that this should always be ASCII code. Still concerned that this is not EBCDIC-safe.